force-app/main/default/classes/MyMethods.cls

```diff
@@ -1,59 +1,39 @@
 public with sharing class MyMethods {
-    
-    /**
-     * Given a string, return its backwards order.
-     */
-    public static String reverseString(String s) {
-        String stringReversed = '';
-
-        for (Integer i = s.length() - 1; i >= 0; --i) {
-            stringReversed += s.substring(i, i + 1);
-        }
-
-        return stringReversed;
-
-
     /**
-     * Given a string, determine if it is a palindrome.
-     */
-    public static Boolean isPalindrome(String s) {
-        s = s.toLowerCase();
-
-        Integer i = 0;
-        Integer j = s.length() - 1;
-
-        while (i < j && s.charAt(i) == s.charAt(j)) {
-            ++i;
-            --j;
-        }
-        return (i >= j);
-
+     *  Given a state, return the accounts in that state.
+     *  I only want the accounts that have a contact who works at the same place as the account.
+     * */
+    public static List<Account> getAccountsForAGivenState(State s) {
+        List<Account> accountList = [
+            SELECT 
+                Id 
+            FROM Account 
+            WHERE BillingState =: s.name()];
+
+        List<Contact> contactList = [
```

Comment:
Currently, this grabs all Contacts in the state regardless of if they are assigned to an Account. If you add `WHERE AccountId != null` to the query, it'll help reduce the contacts to more-relevant ones.